gha/unity/action.yml

```diff
@@ -46,33 +42,26 @@ runs:
             --version ${{ inputs.version }} \
             --logfile "testapps/release_license.log"
           cat testapps/release_license.log
-    - name: Install Unity installer (U3D)
-      if: inputs.release_license == ''
-      uses: nick-invision/retry@v2
-      with:
-        timeout_minutes: 10
-        max_attempts: 2
-        shell: bash
-        command: gem install u3d -v 1.2.3
-    - name: Set the Unity Env
+    - name: unity_setting
```

Comment:
Doesn't need to be done now, but after spending some time with this action, I think this should be split into 3 separate ones.
1. Install
2. Activate
3. Deactivate

Doing it based on input parameters is a bit confusing, especially at a glance.

Reply:
I just want to make the usage simpler: https://github.com/firebase/firebase-unity-sdk/tree/main/gha/unity#usage
But I will add more variables in this action and make the action more descriptive.